src/purifier/package.json

```diff
@@ -7,11 +7,19 @@
   "main": "dist/purifier.js",
   "module": "dist/purifier.mjs",
   "types": "purifier.d.ts",
```

Comment:
For some reason we publish this package to [@ampproject/purifier](https://www.npmjs.com/package/@ampproject/purifier) even though it isn't used by anyone. Our code directly references the relevant files via relative filepaths instead of via package name.

Should we also bump version + republish? With 0 consumers I guess it really doesn't matter :)

Reply:
If removing the package will substantially lower the maintenance burden ([these PRs](https://github.com/ampproject/amphtml/pulls?q=is%3Apr+author%3Arenovate-bot+purifier+is%3Aclosed)), I'd be happy to abandon this PR and let you clean up the publishing stuff.

Reply:
`npm deprecate`?

Reply:
I didn't mean that we shouldn't merge this PR!
But this is my first time noticing that we actually _publish_ the package too, which seems silly.

Merge away!

Reply:
I've deprecated the `@ampproject/purifier` package on npm. Cleanup PR coming up.